src/calibrate/jobs/MUK-BAM.csv

```diff
@@ -0,0 +1,1262 @@
+Time,ConcRT(ug/m3),ConcHR(ug/m3),ConcS(ug/m3),Flow(lpm),WS(m/s),WD(Deg),AT(C),RH(%),BP(mmHg),FT(C),FRH(%),Status
```

Comment:
- In the near future, could we explore alternative ways of storing and accessing the data in these csv files?
- How frequent are these files updated?

Reply:
Thanks, will look into it.
Ideally, the data should come from the database, but I think it's not there yet. For BAM, low-cost and mobile devices. Are we streaming BAM data yet?